src/firmware/linux/host/types/snp.rs

```diff
@@ -211,34 +211,61 @@ impl CertTableEntry {
     }
 }
 
-#[repr(C)]
-pub struct SnpSetExtConfig {
-    /// Address of the Config or 0 when reported_tcb does not need
-    /// to be updated.
-    pub config_address: u64,
-
-    /// Address of extended guest request [`CertTableEntry`] buffer or 0 when
-    /// previous certificate(s) should be removed via SNP_SET_EXT_CONFIG.
-    pub certs_address: u64,
+/// SNP_COMMIT structure
+#[cfg(feature = "snp")]
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Default)]
+#[repr(C,packed)]
+pub struct SnpCommit{
+    /// len of the command buffer read by the PSP
+    length: u32
+}
```

Comment:
Again, may be worth looking at doing this?
```suggestion
/// SNP_COMMIT structure
/// - length: length of the command buffer read by the PSP
#[cfg(feature = "snp")]
#[derive(Copy, Clone, Debug, PartialEq, Eq, Default)]
#[repr(C,packed)]
pub struct SnpCommit(u32);
```